examples/cli/src/main/java/io/unitycatalog/cli/CatalogCli.java

```diff
@@ -54,7 +54,12 @@ public static void handle(CommandLine cmd, ApiClient apiClient) throws JsonProce
 
     private static String createCatalog(CatalogsApi catalogsApi, JSONObject json) throws JsonProcessingException, ApiException {
         CreateCatalog createCatalog;
-        createCatalog = objectMapper.readValue(json.toString(), CreateCatalog.class);
+
+        JSONObject jsonWithoutProperties = new JSONObject(json, JSONObject.getNames(json));
+        jsonWithoutProperties.remove("properties");
+
+        createCatalog = objectMapper.readValue(jsonWithoutProperties.toString(), CreateCatalog.class).properties(CliUtils.extractProperties(objectMapper, json));
+        
```

Comment:
While I agree that this was the conventional way of doing this, it will become confusing as we add more fields, let us instead create a `CreateCatalog` object and assign each field like [here](https://github.com/unitycatalog/unitycatalog/blob/f7c88815d2d70781d595979c1b0f6224d06d5f66/examples/cli/src/main/java/io/unitycatalog/cli/TableCli.java#L81) for TableCli